scrapers/LingerieTales.yml

```diff
@@ -0,0 +1,29 @@
+name: LingerieTales
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - lingerietales.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title:
+        selector: //h1[@class="title"]
+      Date:
+        selector: //time
+        postProcess:
+          - parseDate: January 2, 2006
+      Performers:
+        Name: //ul[@class='filters_buttons']/li[2]/a
+      Tags:
+        Name:
+          selector: //ul[@class="filters_buttons"]/li/a
```

Comment:
the tags include the performers
i think we should filter the href from the a 
```yaml
      Tags:
        Name: //ul[@class="filters_buttons"]/li/a[contains(@href,"/videotag/")]
```
and maybe use the same for performers?
```yaml
      Performers:
        Name: //ul[@class="filters_buttons"]/li/a[contains(@href,"/model-item/")]
```